boards/arm/atsamr21_xpro/pinmux.c

```diff
@@ -95,6 +95,11 @@ static int board_pinmux_init(struct device *dev)
 #warning Pin mapping may not be configured
 #endif
 
+#if (ATMEL_SAM0_DT_TCC_CHECK(0, atmel_sam0_tcc_pwm) && CONFIG_PWM_SAM0_TCC)
```

Comment:
nit: change this to

#if ATMEL_SAM0_DT_TCC_CHECK(0, atmel_sam0_tcc_pwm) && defined(CONFIG_PWM_SAM0_TCC)

Also feel free to skip this as we can do this as a bulk update to all of the related pinmux files.

Reply:
@nzmichaelh sure! Did you open an issue to do that?